babel.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  presets: ['module:metro-react-native-babel-preset'],
+  presets: ['module:@react-native/babel-preset'],
```

Comment:
`metro-react-native-babel-preset` is no longer maintained. It has been migrated to `@react-native/babel-preset` package.